extensions/scarb-cairo-test/src/main.rs

```diff
@@ -43,6 +44,7 @@ fn main() -> Result<()> {
     let metadata = MetadataCommand::new().inherit_stderr().exec()?;
 
     check_scarb_version(&metadata);
+    check_cairo_test_plugin(&metadata);
```

Comment:
when will this be shown?
I suppose it should only when someone tries to execute a test and does not have any plugin configured? or what is the deal?

Reply:
Only if you run `cairo-test` (this is the main file of the cairo-test binary distributed with Scarb) and you miss the plugin.

Reply:
aaaah, sorry misread the file name. thanks for pointing this out :+1: